pkg/thing/interactors/update_data.go

```diff
@@ -39,32 +39,33 @@ func (i *ThingInteractor) verifyThingData(authorization, thingID string, data []
 		return fmt.Errorf("error getting thing metadata: %w", err)
 	}
 
-	if thing.Schema == nil {
+	if thing.Config == nil {
+		//TODO Update error
 		return ErrSchemaUndefined
```

Comment:
Should this error be renamed to `ErrConfigUndefined`?

Reply:
Done!